src/pages/settings/Troubleshoot/TroubleshootPage.tsx

```diff
@@ -81,6 +83,11 @@ function TroubleshootPage({shouldStoreLogs}: TroubleshootPageProps) {
             .reverse();
     }, [shouldStoreLogs, translate, waitForNavigate, styles.sectionMenuItemTopDescription]);
 
+    const isNativePlatform = useMemo(() => {
+        const platform = getPlatform();
+        return platform === CONST.PLATFORM.IOS || platform === CONST.PLATFORM.ANDROID;
+    }, []);
```

Comment:
I'm OK platform specific, but I think we should have this function in a separate file as the guidelines say here https://github.com/Expensify/App?tab=readme-ov-file#platform-specific-file-extensions 

We can use `getTripIllustrationStyle` as an example.